File: Golang Interview QnA/Infosys Interview/infosys.py
class Main:
    def function_name():
        char=['a','a','b','b','c','d']
        seen={}

        for i,c in enumerate(char):
            if c not in seen:
                seen[c]=1
                print(c+str(seen[c]),end="")
            else:
                seen[c]+=1
                print(c+str(seen[c]),end="")
            if i!=len(char)-1:
                print(",",end="")
                
                
Main.function_name()

File: Golang Interview QnA/Infosys Interview/infosys.go
package main

import "fmt"

func main() {
	char := []string{"a", "a", "b", "b", "c", "d"}
	seen := make(map[string]int)
	for i, c := range char {
		if _, exists := seen[c]; exists {
			seen[c]++
			fmt.Print(c, seen[c])
		} else {
			seen[c] = 1
			fmt.Print(c, seen[c])
		}
		if i != len(char)-1 {
			fmt.Print(",")
		}
	}
}
